src/add-and-remove.js

```diff
@@ -0,0 +1 @@
+console.log('next milestone');
```

Comment:
- [ ] Hi, Please ensure to place the functionality for adding and removing into this add-and-remove.js file. please remove the code from index.js and place the new functionality here.